package.json

```diff
@@ -13,7 +13,7 @@
     "url" : "http://github.com/victorops/hubot-victorops.git"
   },
   "dependencies": {
-    "ws": "=0.4.31"
+    "ws": "=4.1.0"
```

Comment:
thats a lot of progress

Reply:
One might even be tempted to say exponential.